src/main/java/frc/robot/RobotManager.java

```diff
@@ -69,6 +72,8 @@ public void teleopInit() {
 
 	@Override
 	public void robotPeriodic() {
+		roborioCycles++;
+		Logger.recordOutput("RoborioCycles", roborioCycles);
 		TimeUtils.updateCycleTime(); // Better to be first
```

Comment:
why is there a comment how this line should be first, but it isn't first.
Either remove the comment if it does not need to be the first statement in the function, or move this line to be the first